src/Orleans/Configuration/GlobalConfiguration.cs

```diff
@@ -593,6 +599,8 @@ internal GlobalConfiguration()
 
             ProviderConfigurations = new Dictionary<string, ProviderCategoryConfiguration>();
             GrainServiceConfigurations = new GrainServiceConfigurations();
+            DefaultVersionCompatibilityStrategy = BackwardCompatible.Singleton;
+			DefaultVersionPlacementStrategy = MinimumVersionPlacement.Singleton;
```

Comment:
nit: there is weird indenting here and in the property declaration.
UPDATE: TABS!! don't want to start a religious war about tabs vs spaces, since I don't care, but I care about consistency, so let's continue to use spaces.